src/mood.ts

```diff
@@ -0,0 +1,139 @@
+import { cliExecute, equip, getClanRumpus, myPrimestat, useSkill } from "kolmafia";
+import { $effect, $effects, $item, $skill, $stat, get, have, Mood, Witchess } from "libram";
+
+Mood.setDefaultOptions({
+  songSlots: [
+    $effects`The Sonata of Sneakiness, Carlweather's Cantata of Confrontation`,
+    $effects`Fat Leon's Phat Loot Lyric`,
+    $effects`Chorale of Companionship`,
+    $effects`Ode to Booze, Ur-Kel's Aria of Annoyance`,
+  ],
+  reserveMp: 100,
+});
+
+export function moodBaseline(): Mood {
+  const mood = new Mood();
+
+  // Combat.
+  mood.skill($skill`Carol of the Hells`);
+
+  // Elemental res.
+  mood.skill($skill`Elemental Saucesphere`);
+  mood.skill($skill`Astral Shell`);
+
+  // Misc.
+  mood.skill($skill`Fat Leon's Phat Loot Lyric`);
+  mood.skill($skill`Singer's Faithful Ocelot`);
+  mood.skill($skill`Blood Bond`);
+  mood.skill($skill`Empathy of the Newt`);
+  mood.skill($skill`Leash of Linguini`);
+
+  return mood;
+}
+
+export function moodNoncombat(): Mood {
+  const mood = moodBaseline();
+
+  mood.skill($skill`The Sonata of Sneakiness`);
+  mood.skill($skill`Smooth Movement`);
+
+  if (get("horseryAvailable") && get("_horsery") !== "dark horse") {
```

Comment:
Do we ever adventure in +combat zones during a casual? I'm pretty sure the answer is no, but be certain of that before you commit to this

Reply:
nope, and even if we do equipment should get you to +25